### ISSUE TITLE
CE_loss in validation data

### ISSUE THREAD

**Issue by jack-gits** [bubbliiiing/facenet-pytorch#5]
![image](https://user-images.githubusercontent.com/30545972/159865496-a3e732fd-21a4-46e7-bbe0-213446f03784.png)

验证集的CE loss比较大，这个算是正常吗？ 怎么降低验证集的CE loss

**Comment by bubbliiiing** [comment]
…是比较大，是之前的下载的库么？我今天刚push上一个新的

**Comment by bubbliiiing** [comment]
可能都过拟合了

**Comment by jack-gits** [comment]
一个月前下载的吧。
为啥在LFW上验证还算正常。

**Comment by bubbliiiing** [comment]
验证集不是LFW数据集，是另一个

**Comment by bubbliiiing** [comment]
自动化分的

**Comment by jack-gits** [comment]
如果过拟合，在LFW上的测试结果应该也不理想，这样才合理。

**Comment by bubbliiiing** [comment]
也很有可能是全连接部分过拟合…LFW不需要全连接部分的…你的val_loss曲线？

**Comment by jack-gits** [comment]
val loss还在下降，但下降很慢。

**Comment by bubbliiiing** [comment]
可以试试新push的代码
没有上升就没过拟合，过去的数据增强方式不太合理其实。

**Comment by jack-gits** [comment]
你一般训练多少epoch可以达到一个满意结果？ 100？

**Comment by bubbliiiing** [comment]
emm……貌似现在50就可以，你可以试一下

**Comment by bubbliiiing** [comment]
因为数据就40W，比较少

**Comment by bubbliiiing** [comment]
还有个arcface也是50

**Comment by jack-gits** [comment]
好的，谢谢了

**Comment by bubbliiiing** [comment]
新上传这个没测试的很详尽， 可以试试100先

**Comment by bubbliiiing** [comment]
暂时没机子测，就简单试了试效果

**Comment by bubbliiiing** [comment]
我看是正常收敛的
